requirements.txt

```diff
@@ -21,3 +21,4 @@ PyYAML==3.11
 pint>=0.7
 numpy
 functools32
+elasticsearch>=2.0.0,<3.0.0
```

Comment:
Thanks! One question for the comment. 
Would it be better to also add a separate requirement file, i.e. elasticsearch-requirement.txt ?